src/reader.js

```diff
@@ -148,8 +148,12 @@ class Reader extends EventEmitter {
 
     // Trigger a query of the configured `lookupdHTTPAddresses`.
     const endpoint = this.roundrobinLookupd.next();
-    lookup(endpoint, this.topic, (err, nodes = []) =>
-      nodes.map(n => this.connectToNSQD(n.broadcast_address, n.tcp_port)));
+    lookup(endpoint, this.topic, (err, nodes = []) => {
+      if(err){
+        this.emit(Reader.ERROR, err);
+      }
```

Comment:
I could have sworn I left this comment previous, but here it goes again :) I don't think it's a good idea to produce an error for a lookup error. The `Reader` shouldn't die just because of a lookup error. Can you help me understand what issue this fixes?